security-council-rehearsals/step-1/README.md

```diff
@@ -0,0 +1,268 @@
+# Rehearsal 1 - Welcome to SuperchainOps
+
+## Objective
+
+This rehearsal is intended simply to ensure that all the signers feel
+confident running the tooling and performing the validations required
+to execute an onchain action.
+
+Once completed, you can be sure that if the task is executed, the WelcomeToSuperchainOps contract will return `"Welcome to SuperchainOps, <name>"` from its [`welcome()`](https://etherscan.io/address/0x0fb11b4164894912f079de62699f4cc5a2271f0c#readContract#F2) method.
+
+The call that will be executed can be found in the `build` function of the `WelcomeToSuperchainOps` template.
```

Comment:
Please include a link to the template. Also, why is the template not in `security-council-rehearsals/templates`?

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/1128/commits/e86266f87a3dc0b79fba557f09ff65824f40eb3f